lib/rubber/cloud/digital_ocean.rb

```diff
@@ -115,6 +137,25 @@ def describe_instances(instance_id=nil)
       def active_state
         'active'
       end
+
+      def destroy_instance(instance_id)
+        # The Digital Ocean API will return a 422 if we attempt to destroy an
+        # instance that's in the middle of booting up, so wait until it's
+        # in a non-"new" state
+        print "Waiting for non-new instance state"
+
+        loop do
+          instance = describe_instances(instance_id).first
+
+          print "."
+
+          break unless instance[:state] == "new"
```

Comment:
Minor style nitpick, but we generally use single-quoted strings if interpolation isn't being used.